versions.props

```diff
@@ -1,2 +1,2 @@
-com.palantir.conjure.python:conjure-python = 3.17.0
+com.palantir.conjure.python:conjure-python = 4.4.0
```

Comment:
The bump to 4.0 was for using python 3.8 https://github.com/palantir/conjure-python/releases/tag/4.0.0 but we've been [using 3.8 since March 2](https://github.com/palantir/conjure-python-client/pull/111)